test/web-auth/web-auth.test.js

```diff
@@ -21,7 +20,16 @@ var IframeHandler = require('../../src/helper/iframe-handler');
 var objectHelper = require('../../src/helper/object');
 var WebAuth = require('../../src/web-auth');
 
+function restoreAndStubStoredTransaction(expectedState, expectedTransaction) {
+  TransactionManager.prototype.getStoredTransaction.restore();
+  stub(TransactionManager.prototype, 'getStoredTransaction', function(state) {
+    expect(state).to.be(expectedState);
+    return expectedTransaction;
+  });
+}
+
 describe('auth0.WebAuth', function() {
+  this.timeout(5000);
```

Comment:
Maybe that request should be mocked and done "locally". We should catch any other errors in the future "integration tests" base that runs against real API

Reply:
While I do agree this would be better, I'm not changing all the tests that do token validation right now. I might do that later if we think it's worth it

Reply:
Can you please add an issue to track this discussion?

Reply:
I added an internal issue.